src/redux/reducers/index.ts

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import net from './net';
-import transactions from './transactions'
+import transactions from './transactions';
+import notifications from './notifications';
 
 export default combineReducers({
   net,
-  transactions
+  transactions,
+  notifications,
```

Comment:
overall code looks good, but is this reducer actually even used anywhere? maybe we can just remove it if toastify is handling everything for us?